modules/platforms/python/cpp_module/utils.cpp

```diff
@@ -47,6 +49,12 @@ bool check_errors(ignite::diagnosable& diag) {
             break;
 
         case SQL_ERROR:
+        default:
+            if (records.get_status_records_number() == 0) {
+                err_msg = "Unknown error";
```

Comment:
Can we provide a better message?

Reply:
In this case - no, unfortunately. But the good part that this should actually never happen.